src/actions/BlogActions.js

```diff
@@ -0,0 +1,5 @@
+export const ADD_BLOG = 'ADD_BLOG';
+export const addBlog = blog => ({
```

Comment:
you're going to want to integrate these instructions from the rubric — "posts have an title and body".

So you'll probably want to deconstruct blog for your payload. ie export const addBlog = ({ title, body }) => 

... and then you'll wan to change your payload with that in mind. :)